onnxruntime/core/optimizer/pad_fusion.cc

```diff
@@ -28,6 +28,7 @@ bool PadFusion::SatisfyCondition(const Graph& graph, const Node& node, const log
 
   const Node& child_node = *node.OutputNodesBegin();
   if (!graph_utils::IsSupportedOptypeVersionAndDomain(child_node, "Conv", {1, 11}) &&
+      !graph_utils::IsSupportedOptypeVersionAndDomain(child_node, "AveragePool", {1, 7, 10, 11, 19}) &&
```

Comment:
JFYI there will be an AveragePool 22 as well upcoming.

Reply:
Yeah, for now to unblock current use case, I went with bit conservative approach because version 22 has big delta which might require more change to the fusion.